gamemode/client/hotkeys.lua

```diff
@@ -33,6 +33,16 @@ hook.Add("KeyPress", "PressShiftHunterHintUpdown", function(ply, key)
     end
 end)
 
+hook.Add("PlayerButtonDown", "PressFToPayRespects", function(ply, button)
+    if (CLIENT and
+        IsFirstTimePredicted() and
+        button == KEY_F
+    ) then
+        net.Start("Pay Respects")
+        net.SendToServer()
+    end
+end)
```

Comment:
Can this be a server-side hook? A client-side hook that sends the server a message screams "server-side hook" to me.

Reply:
whoops lol I just assumed a server wouldn't have a hook for a client event

yeah that def sounds like the right thing, but I'm not going to have a chance to change it before tonight 😔 feel free to upload an edit or I can clean it up later

Reply:
this despite our discussion that would have made it clear if I'd thought about it for two seconds...

Reply:
I modelled this poorly, I didn't realized some of these hooks worked in the SERVER context as well. Why don't we merge as is and we'll do a cleanup later.